src/components/RequestInfo.tsx

```diff
@@ -0,0 +1,317 @@
+                    <div
+                        key={item.label}
+                        className="flex flex-1 flex-col items-center gap-16 rounded-2xl border border-[#E4E9F4] bg-white py-12 px-10"
+                    >
+                        <div>
+                            <div className="mb-6 text-2xl font-semibold leading-[31.2px] tracking-tight text-gray-500">
+                                {item.label}
+                            </div>
+                            <div className="mb-8 text-xl leading-[26px] tracking-tight text-gray-500">
+                                {item.description}
+                            </div>
+                            <ReadCaseStudyLink
+                                parentClassName="text-left"
+                                linkClassName="btn btn-link btn-link-icon lg:focus:ring-gray-300 p-0 text-right text-violet-500 lg:text-gray-700 font-semibold tracking-normal lg:mx-0 lg:text-left"
+                                href={item.href}
+                                linkLabel={item.linkLabel}
+                                openNewTab={true}
+                            />
+                        </div>
+                    </div>
+                ))}
+            </div>
+        </ContentSection>
+    )
+}
+
+const CaseStudySection: FunctionComponent = () => (
+    <div className="flex flex-col items-center gap-y-8 pt-16 lg:flex-row lg:flex-row lg:gap-y-0 lg:gap-x-5 lg:pt-0">
+        <img src="/assets/users-love-us.svg" className="h-[135px] w-[104px]" />
+        <div className="sg-contact-sales rounded-2xl border border-[#E4E9F4] py-10 px-8">
+            <div className="flex flex-col">
+                <div className="mb-6 flex flex-col">
+                    <span className="text-base leading-6 text-gray-500">Brendan Doyle</span>
+                    <span className="text-sm leading-[21px] tracking-normal text-gray-700">
+                        Senior Software Engineer, Qualtrics
+                    </span>
+                </div>
+                <div className="text-xl leading-[26px] tracking-tight text-gray-700">
+                    “Something that would've taken me multiple dev days was done in an hour with Cody.”
+                </div>
+                <div>
+                    <ReadCaseStudyLink
+                        openNewTab={true}
+                        parentClassName="text-right pt-10"
+                        linkClassName="btn btn-link btn-link-icon p-0 text-right text-gray-700 font-semibold !tracking-tight lg:mx-0 lg:text-left focus:ring-gray-300"
+                        href="https://sourcegraph.com/case-studies/qualtrics-speeds-up-unit-tests-and-code-understanding-with-cody"
+                    />
+                </div>
+            </div>
+        </div>
+    </div>
+)
+
+const HeaderListTabItems = [
+    {
+        icon: <Timer className="text-blurple-400" />,
+        header: 'Increase developer productivity',
+        listItems: [
```

Comment:
This list items are not wrapping correctly on mobile. When they go to two lines, the 2nd line should be left-aligned with the start of their 1st line, not the bullet point. 
<img width="558" alt="Screenshot 2024-10-14 at 11 16 43 AM" src="https://github.com/user-attachments/assets/a97164de-332f-456a-8e8d-8925d3bdacfe">